ddl/ingest/backend_mgr.go

```diff
@@ -77,23 +72,17 @@ func (m *backendCtxManager) Register(ctx context.Context, unique bool, jobID int
 	return bc, nil
 }
 
-func createLocalBackend(ctx context.Context, cfg *Config, glue glue.Glue) (*local.Backend, error) {
+// CreateLocalBackend creates a lightning local backend.
+func CreateLocalBackend(ctx context.Context, cfg *Config) (*local.Backend, error) {
 	tls, err := cfg.Lightning.ToTLS()
 	if err != nil {
 		logutil.BgLogger().Error(LitErrCreateBackendFail, zap.Error(err))
 		return nil, err
 	}
 
-	logutil.BgLogger().Info("[ddl-ingest] create local backend for adding index", zap.String("keyspaceName", cfg.KeyspaceName))
```

Comment:
better to require the reviewer of DDL if we change their log

Reply:
ok